gradle.properties

```diff
@@ -1,6 +1,6 @@
 springVersion=4.1.1.RELEASE
 awsJavaSdkVersion=1.11.6
 jacksonVersion=2.7.4
-jerseyVersion=2.22.2
+jerseyVersion=2.23
```

Comment:
If we are to update Jersey, please use the latest version 2.24.1.
On another matter, I dimly recall I ran into an issue that logs for service tests vanished when updating to Jersey 2.23, something to do with some extra configuration needed. This is why I went to Jersey 2.22.2 rather than 2.23 (the latest available at the time). Can you please check that Jersey logs are still present when running service tests.